Common/Sources/View/Error/ErrorView.swift

```diff
@@ -76,10 +76,20 @@ public final class ErrorView {
     }
 
     private func unexpectedError() {
+      if let pageName = self.pageName, let pageGroup = self.pageGroup {
+       let errorInfo = ErrorInfoBuilder(type: self.errorTypeException,
+                                        message: self.error.errorDescription ?? "")
+                .fieldName(fieldName: "")
```

Comment:
Missing `code` - should be `code: error.getHyperwalletErrors()?.errorList?.first?.code`

Reply:
Code modified